### ISSUE TITLE
Call for nominations still active

### ISSUE THREAD

**Issue by goldfirere** [haskellfoundation/haskellfoundation.github.io#92]
I was surprised to discover that https://haskell.foundation/board-nominations/ is still a live page.

It's actually useful to keep this info around, in order to answer how we were initially formed. But the phrasing on the page is much more current than that!

**Comment by myShoggoth** [comment]
Agreed, some language to say that this page is retained for historical context would be useful.
